client/app/visualizations/chart/index.js

```diff
@@ -53,9 +56,13 @@ function ChartEditor(ColorPalette, clientConfig) {
 
       scope.stackingOptions = {
         Disabled: null,
-        Enabled: 'normal',
-        Percent: 'percent',
+        Stack: 'stack',
+        Overlay: 'overlay',
+        Group: 'group',
+        Relative: 'relative',
```

Comment:
I couldn't see the difference between `Group` and `Disabled` or `Relative` and `Stack`. At least not with bar charts (maybe it works different with other chart types?).

Unless it's very important let's keep those settings (and any other) as they were to ease the rollout of this code, as otherwise we will need to start applying data migrations which is better to avoid in this case.

Reply:
Do your data have negative values? With negative values all combinations have clear difference

Reply:
OK, now I can see the difference between Relative and Stack. But still not between group and disabled.

Why not always use relative mode when stacking and then we don't need a different option?

Reply:
All that modes really depends on data:

![image](https://user-images.githubusercontent.com/12139186/34486713-b3fbc65e-efd9-11e7-9aa3-37459b89029d.png)

![image](https://user-images.githubusercontent.com/12139186/34486717-be108d78-efd9-11e7-94d8-08c05537f40a.png)

About relative mode: we cannot always use it, because it displays data in different way: instead of absolute values, it normalizes values against sum, so range of y-axis will be 0..100 (%). And relative mode is not a stacking option itself, it's an option to prepare data before displaying.

Reply:
If it's just the way the data prepared, just the more reason not to have it as an option in the UI :)

The end result should be: the user has three options: grouped, stacked or percent stacked. They shouldn't need to care about how we achieve this, we should just show it correctly (within reason) :)

In the first screenshot above it looks super weird because of two Y axes, right? I'm not sure this is a huge use case to be concerned about. But don't think we need to stack across Y axes (if it's something we can control).